puzzlejs/puzzle.js

```diff
@@ -1780,6 +1836,8 @@ function PuzzleGrid(puzzleEntry, options, container, puzzleId) {
         spokes[l] = spoke;
     }
     var extracts = puzzleEntry.getOptionArray(this.options, "data-extracts", " ");
+    var links = puzzleEntry.getOptionArray(this.options, "data-links", /\s+/);
```

Comment:
Missed this before - I think this should be a single space like the line above, instead of a regexp for any number of spaces. Reason being that consecutive spaces are the way you choose to skip labeling a square.

Reply:
Thanks! I agree that a different separator might someday be valuable, but this is the one that made sense at the time for clues etc. and I'd like to be consistent.